src/index/ivf_raft/ivf_raft_config.h

```diff
@@ -26,6 +26,7 @@ namespace knowhere {
 class RaftIvfFlatConfig : public IvfFlatConfig {
  public:
     CFG_LIST gpu_ids;
+    CFG_INT streams_per_device;
```

Comment:
Ideally, this would be a application-wide configuration rather than a per-index configuration, since only the first value will be used. I wasn't sure if there was a straightforward way to set this at the application level, though.

Reply:
Also, we should document this so that users understand the tradeoffs of setting this value higher. Higher number of streams per device may allow for higher throughput (depending on the size of the index and the batch size), but it will also lead to higher device memory usage, especially if the server is receiving many simultaneous requests.

Reply:
This is a great job. However, the option option in config, is a json from the search task. streams_per_device should be a system configuration option, so my suggestion is that we can use environment variables or open a setup api.

Reply:
The original version was per-stream per-thread, and the simplification may also be an advantage.

Reply:
Great! Do we have a preference between using environment variables vs a setup API? We can handle the environment variable the exact same way you did the environment variable for pool size. If you'd rather use the setup API, though, could you point me to an example of how that's used elsewhere?